gammapy/scripts/config/schema.yaml

```diff
@@ -229,36 +279,34 @@ properties:
 #                unit: TeV
 #                interp: log
 #
```

Comment:
as `geom` is not a main section of the settings anymore, shouldn't we integrate this helping comments block into the `reduction` block somehow?

Reply:
Done